### ISSUE TITLE
Inference on cityscapes dataset, but get a result different from yours

### ISSUE THREAD

**Issue by yamengxi** [ekzhang/fastseg#6]
I use the pretrained model: https://github.com/ekzhang/fastseg/releases/download/v0.1-weights/mobilev3large-lraspp-f128-9cbabfde.pt and inference on cityscapes `val` set.
I get a result with mIOU 85.62, but in your [statement](https://github.com/ekzhang/fastseg#pretrained-models-and-metrics), the mIOU is 72.3, there is a 13.32 gap.
What happened?

**Comment by ekzhang** [comment]
Hi @yamengxi, I believe my validation set was with a different validation split after taking all of the fine-labeled data. So probably the reason why you're seeing such high mIOU is because you actually tried to validate the model on the training data. Sorry for any confusion.

The script I used to train/validate the model is in https://github.com/ekzhang/semantic-segmentation. It uses cv split 2 from the NVIDIA repository. See [`datasets/cityscapes.py`](https://github.com/ekzhang/semantic-segmentation/blob/main/datasets/cityscapes.py).

**Comment by yamengxi** [comment]
I evaluated as what you said and get a result with mIOU **72.12**, this result is quite close to your report **72.3**.
But why I can not train to get such high points?
